### ISSUE TITLE
NTR: Recombinant JCV-Virus protein 1 antibody titer high in CSF (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8743]
New term request Recombinant JCV-Virus protein 1 antibody titer high in CSF (samant_110123155513): VP1; cerebrospinal fluid; Creutzfeldt-Jakob disease; Jakob -Creutzfeldt disease

**Comment by pnrobinson** [comment]
@MickeySegal
is it this virus: https://de.wikipedia.org/wiki/JC-Virus
Why "recombinant"?

**Comment by MickeySegal** [comment]
The recombinant part should be deleted - it was probably how the antibodies were obtained, but that is not important.

The Jakob Creutzfeldt part in our description is an error.  JC stands for John Cunningham.

I changed our finding name to JCV-VP1 because otherwise it seemed like the V was listed twice redundantly, and explained the VP1 as virus protein 1 in the synonyms.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000928
- label: Positive CSF JC-virus antibody positivity
- parents: [Positive CSF pathogen-specific antibody positivity (HP:0430139)](https://hpo.jax.org/app/browse/term/HP:0430139)
- pmid: 
- def: The presence of antibodies in the cerebrospinal fluid (CSF) that react against a component of JC-virus (also known as JC-polyomavirus and John Cunningham virus).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8743
- synonyms: Positive CSF JC-polyomavirus antibody positivity(NONE)
